Runtime/Scripts/Internal/FFIClient.cs

```diff
@@ -199,24 +208,26 @@ public FfiResponse SendRequest(FfiRequest request)
         //TODO interface + memory optimisation
         [AOT.MonoPInvokeCallback(typeof(FFICallbackDelegate))]
         static unsafe void FFICallback(IntPtr data, int size)
-        {
+        { 
             #if NO_LIVEKIT_MODE
             return;
             #endif
-            
+
+            if (_isDisposed) return;
             var respData = new Span<byte>(data.ToPointer()!, size);
             var response = FfiEvent.Parser!.ParseFrom(respData);
 
-            Utils.Debug("Callback... " + response.ToString());
             // Run on the main thread, the order of execution is guaranteed by Unity
             // It uses a Queue internally
             Instance.context?.Post((resp) =>
             {
                 var r = resp as FfiEvent;
-                if (r?.MessageCase != FfiEvent.MessageOneofCase.Logs)
-                    Utils.Debug("Callback: " + r?.MessageCase);
+                //if (r?.MessageCase != FfiEvent.MessageOneofCase.Logs)
+                //    Utils.Debug("Callback: " + r?.MessageCase);
```

Comment:
I guess this and other commented logs are still WIP?

Reply:
This one is just very spammy but I wasn't sure if we needed it or not. Thoughts?